---
assurancetourix:
  ros__parameters:
    topic_for_allies_position: /allies_positions_markers
    topic_for_gradient_layer: /ennemies_positions_markers
    camera_settings:
      width: 3840
      height: 2160
      brightness: 29
      contrast: 12
      saturation: 56
      hue: 129
      gamma: 130
      gain: 60
      backlight_compensation: 64
      auto_WB: true
      exposure: 11
    image_post_processing:
      show_image: true
      small_aruco_size: 0.07
      huge_aruco_size: 0.115
    rviz_settings:
      blue_color_ArUco: [0.0, 0.0, 255.0]
      yellow_color_ArUco: [255.0,255.0,0.0]
      default_color_ArUco: [120.0,120.0,120.0]
      arrow_scale: [0.05,0.05,0.05]
      game_elements_scale: [0.07,0.07,0.01]
      robot_type: 2
      game_element_type: 1
      lifetime_sec: 5
      header_frame_id: "assurancetourix"
    simulation:
      robots :
        - ASTERIX
      refresh_frequency: 2
